refarch-integrations/refarch-s3-integration/refarch-s3-integration-service/pom.xml

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>de.muenchen.refarch</groupId>
+        <artifactId>refarch-s3-integration</artifactId>
+        <version>1.1.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>refarch-s3-integration-service</artifactId>
+    <name>refarch-s3-integration-service</name>
+
+    <properties>
+        <apache.commons.io>2.16.1</apache.commons.io>
```

Comment:
Please refactor it in the scheme `<artefact-name.version>`. This is useful to handle dependency updates through Renovatebot. Please check for all dependency version definitions inside all `pom.xml` files.